template/src/index.template.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>{{ title }}</title>
+    <meta charset="utf-8">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="default">
+    <link rel="apple-touch-icon" sizes="120x120" href="/public/logo-120.png">
+    <meta name="viewport" content="width=device-width, initial-scale=1, minimal-ui">
+    <link rel="shortcut icon" sizes="48x48" href="/public/logo-48.png">
+    <meta name="theme-color" content="#ea4235">
+    <link rel="manifest" href="/manifest.json">
```

Comment:
We should always have minimal required og: and tw: meta. Perhaps these live in an include. It's best to use Twitter large summary card so OG image can be reused for Twitter, and include og: img height and width values to precache imagery.
Example project: https://github.com/bigspaceship/sbux-blonde/blob/113d645b563ba8819748e3ffc01e33e9be051569/layouts/main-layout/index.js#L79

Reply:
Add in meta description too. Still think you need keywords even tho Google doesn't use them.

Reply:
I'd also want to build in the anti clickjacking script that @paul-baranowski added from sbux project into his nextjs. 

Should add GA/GTM placeholders too in Google recommended locations.